module/profile/helper.js

```diff
@@ -0,0 +1,84 @@
+const axios = require('axios')
+
+module.exports = class FormsHelper {
+	/**
+	 * Extracts location IDs from the meta information.
+	 * @param {Object} meta - Meta information containing location IDs.
+	 * @returns {Array} - An array of location IDs.
+	 */
+	static async extractLocationIds(meta) {
```

Comment:
@MallanagoudaB you have to add read function here and include the important logic here, then call other functions li ke extractLocationIds if required